examples/standalone-with-org-policies/outputs.tf

```diff
@@ -0,0 +1,100 @@
+  value       = module.secured_data_warehouse.pubsub_writer_service_account_email
+}
+
+output "data_ingestion_bucket_name" {
+  description = "The name of the bucket created for the data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_bucket_name
+}
+
+output "data_ingestion_topic_name" {
+  description = "The topic created for data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_topic_name
+}
+
+output "data_ingestion_bigquery_dataset" {
+  description = "The bigquery dataset created for data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_bigquery_dataset
+}
+
+output "cmek_data_ingestion_crypto_key" {
+  description = "The Customer Managed Crypto Key for the data ingestion crypto boundary."
+  value       = module.secured_data_warehouse.cmek_data_ingestion_crypto_key
+}
+
+output "cmek_bigquery_crypto_key" {
+  description = "The Customer Managed Crypto Key for the BigQuery service."
+  value       = module.secured_data_warehouse.cmek_bigquery_crypto_key
+}
+
+output "cmek_reidentification_crypto_key" {
+  description = "The Customer Managed Crypto Key for the reidentification crypto boundary."
+  value       = module.secured_data_warehouse.cmek_reidentification_crypto_key
+}
+
+output "cmek_confidential_bigquery_crypto_key" {
+  description = "The Customer Managed Crypto Key for the confidential BigQuery service."
+  value       = module.secured_data_warehouse.cmek_confidential_bigquery_crypto_key
+}
+
+output "data_ingestion_access_level_name" {
+  description = "Data Ingestion Access Context Manager access level name."
+  value       = module.secured_data_warehouse.data_ingestion_access_level_name
+}
+
+output "data_ingestion_service_perimeter_name" {
+  description = "Data Ingestion VPC Service Controls service perimeter name."
+  value       = module.secured_data_warehouse.data_ingestion_service_perimeter_name
+}
+
+output "data_governance_access_level_name" {
+  description = "Data Governance Access Context Manager access level name."
+  value       = module.secured_data_warehouse.data_governance_access_level_name
+}
+
+output "data_governance_service_perimeter_name" {
+  description = "Data Governance VPC Service Controls service perimeter name."
+  value       = module.secured_data_warehouse.data_governance_service_perimeter_name
+}
+
+output "confidential_data_access_level_name" {
+  description = "Confidential Data Access Context Manager access level name."
+  value       = module.secured_data_warehouse.confidential_access_level_name
+}
+
+output "confidential_data_service_perimeter_name" {
+  description = "Confidential Data VPC Service Controls service perimeter name"
+  value       = module.secured_data_warehouse.confidential_service_perimeter_name
+}
+
+output "blueprint_type" {
```

Comment:
does this need to be here?

Reply:
We need to test the blueprint_type. In case the blueprint has changed in some invalid way we'll know.